setup.py

```diff
@@ -26,7 +26,12 @@ def run(self, *args, **kwargs):
 
 setup(
     name='azure-functions-durable',
-    packages=find_packages(exclude=("tests", "samples","scripts")),
+    packages=find_packages(exclude=[
+        "tests",
+        "samples",
+        "scripts",
+        "azure"
```

Comment:
Just to be sure: does it matter all of our source code is within a directory inside the "azure" directory of the repo? I would think that this change would essentially exclude all of the Durable Functions code as well :-)

Reply:
@davidmrdavid This change will not exclude all the code inside azure. Removing the init.py from azure would simply make it a namespace instead of a module which is currently is. I had done a similar change to the azure function python library https://github.com/Azure/azure-functions-python-library/pull/97 where we exclude the init file while packaging it.